ion/decimal.go

```diff
@@ -231,21 +231,29 @@ func (d *Decimal) upscale(scale int32) *Decimal {
 	}
 }
 
-// Trunc attempts to truncate this decimal to an int64, dropping any fractional bits.
-func (d *Decimal) Trunc() (int64, error) {
+func (d *Decimal) checkToUpscale() (*Decimal, error) {
```

Comment:
Please add a comment to this method explaining what it means to "upscale" a `Decimal`.

Reply:
Added comment.